### PR TITLE
feat(Tooltip): updated the tooltip to the new design

### PR DESCRIPTION
**Pull request by AlsasaAmmar**
[ONEUI-491](https://textkernel.atlassian.net/jira/software/c/projects/ONEUI/boards/119?assignee=712020%3A63f9d6c8-c4f1-488d-8002-8599ae3602af&assignee=unassigned&selectedIssue=ONEUI-491)

…d make sure that the font of the tooltip is 14px

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. Use imperative, present tense in your commit description ("change", not "changed" or "changes") without uppercases or period (.) at the end.

# Checklist
- [x] The implementation has been manually tested and complies with Textkernel [browser support guidelines](https://textkernel.com/browser-support/)
- [x] The implementation complies with [accessibility](CONTIRBUTING.md#accessibility) standards.
- [x] The component has a [displayName](CONTRIBUTING.md#display-names) defined.
- [x] The component comes with a detailed [PropTypes](CONTRIBUTING.md#component-props) (and defaultProps) definition.
- [x] Component PropTypes are sufficiently described / documented.
- [x] There is [a story](CONTRIBUTING.md#component-showcases) in Storybook.

### CODE REVIEW

File: package-lock.json
Comment (eszthoff) [inline package-lock.json:1]: Did you mean to push these changes? Normally we only do so if there is something changed in package.json, or if the PR is about updating dependencies.

File: src/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /* eslint-disable import/no-extraneous-dependencies, import/no-unresolved */
-import * as React from 'react';
 import Tippy, { TippyProps } from '@tippyjs/react';
-import 'tippy.js/dist/tippy.css?external';
+import * as React from 'react';
```
Comment (eszthoff) [inline src/components/Tooltip/Tooltip.tsx:3]: NIP: why do you move this down? 
Not a hard requirement, as we don't have linting rules for import order, but I think this import is always the first in this codebase.

File: src/components/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ export const Tooltip: React.FC<TooltipProps> = ({ content, children, disabled, .
         isDisabled = true;
     }
     return (
-        <Tippy content={content} disabled={isDisabled} {...rest}>
+        <Tippy content={content} disabled={isDisabled} maxWidth={320} arrow={false} {...rest}>
```
Comment (eszthoff) [inline src/components/Tooltip/Tooltip.tsx:23]: I have difficulty checking in the dev-tools, but it seems to me that the font-size is still large (16px) not medium (14px) as required in the ticket. 

I think you need to make a scss file, and make styles there then apply here. We have a (new) pattern to apply typography. You should use `compose: OneUI-label-text from global`. Let me know if this is not clear and we can look at it together.

File: .gitignore [hunk 3]
```diff
@@ -7,6 +7,7 @@
 ### VisualStudioCode template
 /.vscode
 .vscode/*
+.history
```
Comment (eszthoff) [inline .gitignore:10]: 👍

**Review by eszthoff — CHANGES_REQUESTED** [review]

**Comment by eszthoff** [comment]
we normally don't detail the changes that much in the PR name. You can make it shorter like `feat(Tooltip): apply new design`. We then link the Jira ticket in the description (we should update our template....) so that the details can be found there.

**Comment by eszthoff** [comment]
note that the pipeline failed. I think you need to update the snapshots in the test: `npm test -- -u`

File: src/components/Tooltip/Tooltip.tsx (hunk 2 above)
Comment (AlsasaAmmar) [reply src/components/Tooltip/Tooltip.tsx:23]: image
I did check the font size and it seems to be the way we wanted (14px)

Comment (eszthoff) [reply src/components/Tooltip/Tooltip.tsx:23]: nice! thanks

**Review by eszthoff — APPROVED** [review]

**Pull request merged**